### DISCUSSION TITLE
Adding Rebar for a 2070 Super on B450?

### DISCUSSION THREAD

**Discussion by Artasdmc** [xCuri0/ReBarUEFI#204] [General]
Motherboard already supports it, whats the appropriate way of doing this? As all information in guides speak about intel motherboards.

4G decoding enabled, CSM disabled, REBAR set to auto as the GPU doesn't support it, so there's only 2 options (disabled/auto).

Already tried this guide for Asus motherboards: https://github.com/terminatorul/NvStrapsReBar/discussions/61

EZ-Flash doesn't flash BIOS as it says it's not proper BIOS, probably a signature check.
AFUDOS went through with the flashing apparently, everything succeeded but I think it's all smoke as it's not working and my motherboard profiles are still present. If bios was really flashed they would get deleted.

**Comment by xCuri0** [comment]
Wrong place to ask this.

Your issue is something with the BIOS flashing process.

I think AMD needs old AGESA to flash modified BIOS or something
